test/async-script-loader-spec.js

```diff
@@ -97,40 +93,42 @@ describe("AsyncScriptLoader", () => {
       asyncScriptOnLoadCalled = true;
     }
     const ComponentWrapper = makeAsyncScriptLoader(URL, { globalName: globalName })(MockedComponent);
-    const instance = ReactTestUtils.renderIntoDocument(
+    ReactTestUtils.renderIntoDocument(
       <ComponentWrapper asyncScriptOnLoad={asyncScriptOnLoadSpy} />
     );
 
     assert.equal(hasScript(URL), false, "Url not in document");
     assert.equal(asyncScriptOnLoadCalled, true, "asyncScriptOnLoad callback called");
-    ReactDOM.unmountComponentAtNode(ReactDOM.findDOMNode(instance));
-    instance.componentWillUnmount();
     delete window[globalName];
   });
 
   it("should accept a function for scriptURL", () => {
     const URL = "http://example.com/?url=function";
     const ComponentWrapper = makeAsyncScriptLoader(() => URL)(MockedComponent);
-    const instance = ReactTestUtils.renderIntoDocument(
+    ReactTestUtils.renderIntoDocument(
       <ComponentWrapper />
     );
 
     assert.equal(hasScript(URL), true, "Url in document");
-    ReactDOM.unmountComponentAtNode(ReactDOM.findDOMNode(instance));
-    instance.componentWillUnmount();
   });
 
   it("should expose statics", (done) => {
+    class MockedComponentWithStatic extends React.Component {
+      static callsACallback(fn) { fn(); }
+      render() { return <span/>; }
+    }
     const URL = "http://example.com/?functions=true";
-    const ComponentWrapper = makeAsyncScriptLoader(URL)(MockedComponent);
+    const ComponentWrapper = makeAsyncScriptLoader(URL)(MockedComponentWithStatic);
     ComponentWrapper.callsACallback(done);
   });
 
   it("should not remove tag script on removeOnUnmount option not set", () => {
     const URL = "http://example.com/?removeOnUnmount=notset";
     const ComponentWrapper = makeAsyncScriptLoader(URL)(MockedComponent);
     const instance = ReactTestUtils.renderIntoDocument(
-      <ComponentWrapper />
+      <div>
+        <ComponentWrapper />
+      </div>
```

Comment:
i think `ReactTestUtils.renderIntoDocument` changed a bit with react versions, so we aren't able to just have the `Component` render then call `unmountComponentAtNode`, i needed to wrap in a `div` 🤷‍♂️